source/write-operations/replace-documents.txt

```diff
@@ -0,0 +1,212 @@
+.. _java-rs-write-replace:
```

Comment:
I'm aware of this: i'm making a ticket to do cleanup later and create landing pages/create a toc. The reason why we're doing it in a seperate ticket is so that we can do all the TOC/organizational work in one PR rather than having it be scattered and potentially causing merge conflicts.

Reply:
Sounds good, I will note that for future reviews on this work!